### ISSUE TITLE
Missing compiled binaries for Windows OS in 2.4.0 release

### ISSUE THREAD

**Issue by k-Knight** [Minecraft-Technik-Wiki/BlockMap#74]
## Environment info

- **BlockMap version**: `2.4.0`
- **Java version**: `17.0.2`
- **Operating system**: `Windows`

## Description

Same as title. The binaries for Windows OS are not provided in the 2.4.0 release on GitHub.

**Comment by piegamesde** [comment]
Indeed. I don't have a Windows machine to build on, and the Windows CI went to hell because it was flaky (probably because of a lack of RAM in the runner).

**Comment by saibotk** [comment]
I uploaded the windows jar to the latest release. Enjoy!
